tsconfig.json

```diff
@@ -16,9 +16,14 @@
     "sourceMap": true,
     "strict": true,
     "target": "es2020",
-    "types": ["node", "mocha"]
+    "types": ["node", "mocha", "./typings"]
```

Comment:
question: what does `./typings` bring to the party?

Reply:
removed